pkg/pbq/metrics.go

```diff
@@ -0,0 +1,14 @@
+package pbq
+
+import (
+	metricspkg "github.com/numaproj/numaflow/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+// activePbqsCount is used to indicate the number of tasks in ordered processor
+var activePbqsCount = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
It would be good to add a metric to measure the current PBQ channel buffer size, so we know the PBQ read is able to catch up with the write.

Reply:
sure, will add a metric for that